go/tools/builders/cgo2.go

```diff
@@ -318,6 +318,10 @@ func defaultCFlags(workDir string) []string {
 	flags := []string{
 		"-fdebug-prefix-map=" + abs(".") + "=.",
 		"-fdebug-prefix-map=" + workDir + "=.",
+		"-fcoverage-prefix-map=" + abs(".") + "=.",
```

Comment:
Could you check whether `-ffile-prefix-map` has better support with old GCC versions? Otherwise we would need some kind of feature detection.